vagrant/vagrant/bootstrap_functions.sh

```diff
@@ -35,6 +35,19 @@ function install_utils() {
     cp /vagrant/dot_files/vimrc         /root/.vimrc
 }
 
+function install_python_tools() {
+    #this is used for RED-I2 things
+    #Due to some bug with Python 3.4 and Debian, need to get pip manually
+    curl https://bootstrap.pypa.io/get-pip.py > get-pip.py
```

Comment:
Eventually we will want to cache this thing somewhere we know we can always get it